posts/2022/sdk_11_1_0_rc.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Titanium SDK 11.1.0.RC released'
+date: '2022-08-29'
+category: 'Release'
+author: 'Michael Gangolf'
+teaser: 'The first test version of the next Titanium SDK 11.1.0.RC is available.'
+---
+
+The first test version of the next Titanium SDK is available: <b>11.1.0.RC</b>. Titanium SDK 11.1.0 is a minor release of the SDK, addressing high-priority issues from previous releases.
+
+For a more detailed overview, known issues and closed tickets, see the release notes: [Titanium SDK 11.1.0.RC Release Note](https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_11.x/Titanium_SDK_11.1.0.RC_Release_Note.html).
+
+## Install
+
+**Follow these steps to get SDK 11.1.0.RC:**
+
+1. Install the CLI with `[sudo] npm i -g titanium alloy`
+2. `titanium sdk install 11.1.0.RC`
+
+## Rollback
+
+Execute `titanium sdk install latest` to rollback
```

Comment:
Add `--default`? Set `<sdk-version>` in `tiapp.xml`? Period at the end of the sentence?